Makefile

```diff
@@ -186,7 +186,7 @@ bundle-push:
 # Update index.db in build_deploy script
 .PHONY: index-add
 index-add:
-	 $(OPM) index add --build-tool=docker --bundles $(BUNDLE_IMG) --tag ${INDEX_IMG} -f quay.io/${REG}/observability-operator-index:v${PREV_VERSION} --generate
+	 $(OPM) index add --build-tool=docker --bundles $(BUNDLE_IMG) --tag ${INDEX_IMG} -f quay.io/rhoas/observability-operator-index:v${PREV_VERSION} --generate
```

Comment:
Should this be hardcoded?

Reply:
@StevenTobin Sorry that was just from creating my index. I'll reverse that. Thanks!